input/includes/privacy.md

```diff
@@ -0,0 +1,5 @@
+Due to size constraints and to preserve patient privacy, **information that is not _strictly_ necessary for Verifiers in the context of [this IG's use cases](index.html#use-cases) SHALL NOT be included in SMART Health Cards.**
+
+If private or sensitive information are included in a SMART Health Card, it will be exposed to the Verifier every time the SMART Health Card is presented. Additionally, information cannot be removed without invaliding the entire SMART Health Card.[^explanation] It is therefore critical for Issuers to avoid including any information that could represent a safety or privacy risk to a patient in a SMART Health Card.
```

Comment:
```suggestion
If private or sensitive information are included in a SMART Health Card, it will be exposed to the Verifier every time the SMART Health Card is presented. Additionally, information cannot be removed without invalidating the entire SMART Health Card.[^explanation] It is therefore critical for Issuers to avoid including any information that could represent a safety or privacy risk to a patient in a SMART Health Card.
```